test/e2e/deployment.go

```diff
@@ -226,8 +235,196 @@ func testRollingUpdateDeploymentEvents(f *Framework) {
 			Name: deploymentName,
 		},
 		Spec: extensions.DeploymentSpec{
-			Replicas:       1,
-			Selector:       deploymentPodLabels,
+			Replicas: 1,
+			Selector: deploymentPodLabels,
+			Strategy: extensions.DeploymentStrategy{
+				Type: extensions.RollingUpdateDeploymentStrategyType,
+			},
+			UniqueLabelKey: "deployment.kubernetes.io/podTemplateHash",
```

Comment:
Don't we have this label somewhere as a constant? If not, it should be in pkg/apis/extensions/types.go and any versioned branches (currently only pkg/apis/extensions/v1beta1/types.go)